ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts

```diff
@@ -0,0 +1,21 @@
+export interface IPalindromeChecker {
+  isAPalindrome: (word: string) => boolean
+  getReverseWord: (word: string) => string
+  getWordWithoutSpaces: (word: string) => string
+}
+
+export default class PalindromeChecker implements IPalindromeChecker {
+  
+  getReverseWord(word: string): string {
+    return word.split('').reverse().join('')
+  }
+  
+  getWordWithoutSpaces(word: string): string {
+    return word.replace(/\s/g, '')
+  }
```

Comment:
I'd consider these implementation details and change them to `private` to make the interface of the class narrower.

Reply:
Good idea

Reply:
Ps.: I'd also remove them from `IPalindromeChecker` unless you explicitly want to expose them to the consumers of this API. But then I would call it something else than "PalindromeChecker" since it would have 3 utilities instead of one, maybe "WordUtilities"

Reply:
Yes, I think I should move them from the interface